ast_release.md

```diff
@@ -0,0 +1,48 @@
+# Deploying the OQ3 Python AST
+
+Author: Jake Lishman `jake.lishman@ibm.com`
+
+You need:
+- permission to push tags to `openqasm/openqasm`
+- a reviewer to approve a PR to the branch
+
+Most of the deployment is automated by the GitHub workflows:
+- `/.github/workflows/build-ast.yml`
+- `/.github/workflows/deploy-ast.yml`
+which do what they say on the tin.
+
+The "deploy" workflow uses the organisation-level secret `OPENQASM_BOT_PYPI_TOKEN`, which is an API token for the PyPI user `openqasm-bot`, which I (Jake: `jake.lishman@ibm.com`) currently own.
+PyPI's authorisation policies may need us to change how we handle that in the future, if they bring in some of their proposed changes around API tokens and 2FA.
```

Comment:
PyPI's authorisation policies _did_ change since I wrote this, and now there's the concept of "trusted deployment environments".  We'd still be tied to `openqasm-bot` owning the package to some degree, but I can update the deployment to use the better system (no need to block this PR on that - I'll update it later).